console.lua

```diff
@@ -187,6 +187,73 @@ commands = {{
         end
         return "Discards are now " .. G.GAME.current_round.discards_left
     end
+}, {
+    name = "joker",
+    source = "debugplus",
+    shortDesc = "Spawns a joker",
+    desc = [[Spawns Jokers. Usage:\njoker {args} [name]\n 
+            -a : Skip calling add_to_deck on spawned joker
+            -c : Count of jokers to spawn
+            -e : Edition to spawn jokers with
+            -E : enable jokers spawned to have random editions
+        ]],
+    exec = function(args, rawArgs, dp)
+        if G.STAGE ~= G.STAGES.RUN then
+            return "This command must be run during a run.", "ERROR"
+        end
+        local j_key = nil
+        local j_count = 1
+        
+        local j_no_edition = true
+        local j_set = "Joker"
+        local j_edition = nil
+        local j_rarity = nil
+        local add_to_deck = true
+
+        -- I thought of making a full on arg parser for this command
+        -- that started as a meme so I could spawn 1000 copies of jimbo without using eval
+        local skipNextArg = false
+        for i,v in ipairs(args) do 
+            if skipNextArg then
+                skipNextArg = false
+                -- Don't parse args to other parm
+            elseif not (v:sub(1,1) == "-" ) then
+                j_key = "j_" .. v
+            else
+                skipNextArg = true
+                local param = v:sub(2,2)
+
+                if param == "a" then 
+                    add_to_deck = false
+                    skipNextArg = false
+                elseif param == "E" then 
+                    j_no_edition = false
+                    skipNextArg = false
+                elseif not args[i+1] then 
+                    return "Please specify a value for the argument"
+                end
+                local val = args[i+1] 
+                if param == "c" then 
+                    j_count = tonumber(val) 
+                    if not j_count then
+                        return "Argument to count must be a number"
+                    end
+                elseif param == "e" then j_edition = "e_"..val
+                elseif param == "r" then j_rarity = val
+                -- elseif param == "S" then j_set = val --Bad Idea
+                end
+                
+            end 
+        end
+        local j_skip_zmaterialize = j_count > 5
+        for _=1,j_count do
+            local card = SMODS.create_card{key = j_key, set = j_set, rarity = j_rarity,no_e, j_no_edition}
```

Comment:
DebugPlus has to work without SMODS as a rule of thumb, (unless doing so would be stupidly difficult). As such, I can't rely on SMODS functions. You can use vanilla's create_card. The function signature for it is `function create_card(_type, area, legendary, _rarity, skip_materialize, soulable, forced_key, key_append)`. Unfortunately, there is no edition config, so the random edition toggle and manual edition toggle won't work, without some more work on your end.

Also you have `j_no_edition` in this table without assigning a key to it, so it's just going to be assigned to `1` and ignored.